linkerd/app/src/identity.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::large_enum_variant)]
```

Comment:
slight pref for putting the `allow` on the specific place where the lint fires, if possible, rather than globally for the module — we might add new code that fires the same lint and we might not want to allow that.